charts/loki-distributed/templates/ingester/deployment-ingester.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if eq .Values.ingester.kind "Deployment"}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "loki.ingesterFullname" . }}
+  labels:
+    {{- include "loki.ingesterLabels" . | nindent 4 }}
+    app.kubernetes.io/part-of: memberlist
+spec:
+  replicas: {{ .Values.ingester.replicas }}
+  minReadySeconds: 60 # Should we define this variable or is a static value for the ingesters to register in the ring fine?
```

Comment:
Maybe drop this? And replace it with a liveness and readiness probe. If I take a look at the ingester deployment type in the cortex Helm, 'minreadyseconds' is not defined.

Reply:
there's a readyness probe in place, so i removed the `minReadySeconds`. Didn't check how to do a liveness probe for loki. Since there is none for the statefulset, i assume we don't require it here?